dev-tools/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "require": {
+        "php": "^7.0"
+    },
+    "require-dev": {
+        "maglnet/composer-require-checker": "^0.1.4",
```

Comment:
0.1.5 might be a better match as 0.1.4 is know to be bugged

Reply:
:+1: